CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 *                                   @clarketm @fejta @Katharine @krzyzacy @michelle192837
+/docker/                            @howardjohn @sdake @geeknoid @duderino @linsun
```

Comment:
@clarketm do you want to be in here? `/hold cancel` if not
/cc @clarketm

Reply:
I can PR and add myself in the future if needed.